Src/Support/Google.Apis.Auth/OAuth2/GoogleCredential.cs

```diff
@@ -179,7 +179,7 @@ public Task InterceptAsync(HttpRequestMessage request, CancellationToken taskCan
 
         /// <summary>
         /// <para>Returns <c>true</c> only if this credential type has no scopes by default and requires
-        /// a call to <see cref="o:CreateScoped"/> before use.</para>
+        /// a call to <see cref="CreateScoped"/> before use.</para>
```

Comment:
This needs disambiguating ;)
Error: Ambiguous reference in cref attribute: 'CreateScoped'

Reply:
Whoops - fixed now (have replaced the commit).

Reply:
Still failing ;) It looks like your latest change hasn't arrived in github...

Reply:
Sigh... I think I did `git commit --amend` rather than `git commit -a --amend`. We now have a different hash, so fingers crossed...